src/main/java/frc/robot/subsystems/pivot/PivotIOSim.java

```diff
@@ -30,38 +31,46 @@ public class PivotIOSim implements PivotIO {
   private double followerAppliedVolts = 0.0;
 
   /**
-   * Updates Inputs
+   * Updates inputs for logging w/ advantage kit
    *
    * @param inputs inputs for logging
    */
   @Override
   public void updateInputs(PivotIOInputs inputs) {
-    pivotSim.update(0.02);
+    pivotSim.update(PivotConstants.RIO_FREQUENCY);
 
-    inputs.leaderPosition = (pivotSim.getAngleRads() / (Math.PI * 2)); // Rotations
-    inputs.leaderVelocity = (pivotSim.getVelocityRadPerSec() / (Math.PI * 2)); // Rotations/Sec
+    inputs.leaderPosition = Units.radiansToRotations(pivotSim.getAngleRads());
+    inputs.leaderVelocity = Units.radiansToRotations(pivotSim.getVelocityRadPerSec());
     inputs.leaderAppliedVolts = leaderAppliedVolts;
     inputs.leaderSupplyCurrentAmps = pivotSim.getCurrentDrawAmps();
 
-    inputs.followerPosition = (pivotSim.getAngleRads() / (Math.PI * 2)); // Rotations
-    inputs.followerVelocity = (pivotSim.getVelocityRadPerSec() / (Math.PI * 2)); // Rotations/Sec
+    inputs.followerPosition = Units.radiansToRotations(pivotSim.getAngleRads());
+    inputs.followerVelocity = Units.radiansToRotations(pivotSim.getVelocityRadPerSec());
     inputs.followerAppliedVolts = followerAppliedVolts;
     inputs.followerSupplyCurrentAmps = pivotSim.getCurrentDrawAmps();
   }
 
-  /** Sets Voltage */
+  /**
+   * Sets Voltage of the shooter
+   *
+   * @param volts desired voltage
+   */
   @Override
   public void setVoltage(double volts) {
     leaderAppliedVolts = volts;
     followerAppliedVolts = volts;
     pivotSim.setInputVoltage(volts);
   }
 
+  /**
+   * sets the Pivot angle
+   *
+   * @param angleRots desired angle of shooter in rotations
+   */
   @Override
   public void setPivotAngle(double angleRots) {
-    double angleRads = angleRots * Math.PI * 2;
-    double currentPivotAngleRots = (pivotSim.getAngleRads() / (Math.PI * 2));
-    double armFF = armFeedforward.calculate(angleRads, pivotController.getSetpoint().velocity);
+    double currentPivotAngleRots = Units.radiansToRotations(pivotSim.getAngleRads());
+    double armFF = armFeedforward.calculate(angleRots, pivotController.getSetpoint().velocity);
     setVoltage(pivotController.calculate(angleRots, (currentPivotAngleRots + armFF)));
```

Comment:
`armFF` should be added to the `pivotController.calculate()` not the `currentPivotAngleRots`.

Reply:
done